src/components/Page/components/Header/Header.tsx

```diff
@@ -134,6 +136,9 @@ export function Header({
         actions={secondaryActions}
         groups={actionGroups}
         rollup={isNavigationCollapsed}
+        rollupActionsLabelOverride={i18n.translate(
```

Comment:
Just checking my understanding:

- This ensures all  `Page` `Header` components containing an `ActionGroup` have the same label. 
- This distinguishes them from other `RollupMenu`s that may exist on the page, and provides a consistent experience across the admin 

Is the `ActionMenu` ever used outside the `Page` > `Header` component?

Reply:
> This ensures all Page Header components containing an `ActionGroup` have the same label.

"containing a rolled up `ActionMenu`". But yes your understanding is correct.

I don't see `ActionMenu` used outside of `Page` > `Header` at least within Polaris. But it's a top level component and exported, so I wasn't 100% sure.

Reply:
Thank you 😁 

This is cool idea 🙌  

Assuming all our pages have appropriate headings, the user will know what page they're viewing (reducing the ambiguity of 'page'), and this provides a reliable and consistent way to access the menus across the admin. 

For voice-command users, there's still the issue of discoverability (how do I know to say "View page actions"), but that's going to be an issue anytime we don't have the visual label. 

Possible scope-creep for this PR, but one solution for voice-user discoverability would be to add the label text in a tooltip on hover/focus: https://youtu.be/kMs-mOpWNPk (around the 24minute mark)

I don't know a lot about voice-commands, so I've reached out on the #accessibility channel, just out of curiosity. https://shopify.slack.com/archives/C03T74ZTD/p1616675760070700?thread_ts=1616674601.070600&cid=C03T74ZTD

Reply:
This PR has kinda turned into making the accessibility of the rollup button better so I can look at adding a tooltip 😄